support/fakeclient.go

```diff
@@ -0,0 +1,11 @@
+package support
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+func NewFakeKubeClientWithObjects(objects ...runtime.Object) *fake.Clientset {
```

Comment:
I see a common usage to initialize client using for example:
```
	fakeClient := NewFakeKubeClientWithObjects(fakeIngress...)

	test := With(t).(*T)
	test.client = &testClient{
		core: fakeClient,
	}
```

Wouldn't it have sense to add it here and return `test`?

Reply:
Resolved as per suggestion